tests/manage/z_cluster/nodes/test_automated_recovery_from_failed_nodes_reactive_IPI.py

```diff
@@ -203,12 +203,67 @@ def test_automated_recovery_from_failed_nodes_IPI_reactive(
 @ignore_leftovers
 @tier4
 @tier4a
-@aws_based_platform_required
 @ipi_deployment_required
 class TestAutomatedRecoveryFromStoppedNodes(ManageTest):
 
     osd_worker_node = None
     extra_node = False
+    machineset_name = None
+    start_ready_replica_count = None
+
+    def verify_osd_worker_node_in_ready_state(self, nodes):
+        """
+        Verify that the osd worker node is in ready state.
```

Comment:
This function is not just verifying whether the node status is Ready or not but trying to recover the node to Ready state. Please change the function name accordingly.

Reply:
Okay, that will do.